js/Events/evaluationEvents.js

```diff
@@ -1,16 +1,30 @@
 const EventsEvaluations = function(){
+
     const newEvaluationButton = document.getElementById('newEvaluationButton');
+
     const newButtonListener = function(event){
         event.preventDefault();
         RedirectTo('new');
     };
 
+    const plusButtons = document.querySelectorAll('.image-for-details');
```

Comment:
Here you are using a presentational class (used for the visual / styling) to access items programatically. It is a good practice to not couple the class names like that and to have a specific class you want to use just for the programatical side.

Reply:
Thanks!